string pathToDictionary
---
string emailAddress
byte isFound
